code/game/machinery/vr_console.dm

```diff
@@ -243,3 +248,202 @@
 	else
 		occupant.enter_vr(avatar)
 
+/obj/machinery/vr_sleeper/business
+	var/charge = 15
+	var/paid = FALSE
+	var/owner_name
+	var/datum/money_account/owner_account

```

Comment:
So about this. Unique_save_vars does not store datums - only text and numbers for the sake of sanitation. See what I did with cash registers and display cases - put a bank id here and let the code get_account(bank_id) whenever it needs to reference the actual account.